### PR TITLE
pinning the pyyaml version due to bug.

### PR DESCRIPTION
**Pull request by EliseCastle23**
We were getting the following error due to an issue with pyyaml 5.4:
#12 22.63           raise AttributeError(attr)
#12 22.63       AttributeError: cython_sources
#12 22.63       [end of output]
please see: https://github.com/yaml/pyyaml/issues/724

pinning the pyyaml version until it is resolved.

### CODE REVIEW

File: Docker/awshelper/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,9 @@ RUN apt-get update && apt-get upgrade -y \
       wget \
       gettext-base
 
+#can remove one https://github.com/yaml/pyyaml/issues/724 is solved
```
Comment (k-burt-uch) [inline Docker/awshelper/Dockerfile:41]:
```suggestion
#can remove once https://github.com/yaml/pyyaml/issues/724 is solved
```

**Review by k-burt-uch — DISMISSED** [review]

File: Docker/awshelper/Dockerfile [hunk 2]
```diff
@@ -38,6 +38,9 @@ RUN apt-get update && apt-get upgrade -y \
...
+RUN pip install pyyaml==5.3.1
+
 RUN  python3 -m pip install --upgrade pip \
```
Comment (k-burt-uch) [inline Docker/awshelper/Dockerfile:44]: These run commands aren't going to overwrite the pyyaml installation here, will it?

Comment (EliseCastle23) [reply Docker/awshelper/Dockerfile:44]: Good question! I believe this was what upgraded pyyaml: 
python3 -m pip install --upgrade setuptools

From what I saw online, doing the pip install and pinning a specific version before we run that command will ensure it is not upgraded. This is what I saw locally as well and it appears to have passed :D

Comment (k-burt-uch) [reply Docker/awshelper/Dockerfile:44]: Sounds good. I threw a RUN pip list in the Dockerfile and double checked 5.3.1 is set

```#21 [17/21] RUN pip list
#21 sha256:162d49c60fac4a67a109e9d6dd8243f8f448566422ae41d83e8f66ad03464887
#21 1.661 Package            Version
#21 1.662 ------------------ --------
#21 1.663 aiofiles           0.8.0
#21 1.663 aiohttp            3.8.5
#21 1.664 aiosignal          1.3.1
#21 1.665 anyio              3.7.1
#21 1.665 argcomplete        3.1.1
#21 1.666 async-timeout      4.0.2
#21 1.667 asyncio            3.4.3
#21 1.667 attrs              23.1.0
#21 1.668 backoff            1.11.1
#21 1.668 cdislogging        1.1.1
#21 1.669 certifi            2023.5.7
#21 1.669 charset-normalizer 3.2.0
#21 1.670 click              7.1.2
#21 1.671 crcmod             1.7
#21 1.671 dataclasses-json   0.5.13
#21 1.672 dbus-python        1.2.18
#21 1.672 dictionaryutils    3.0.2
#21 1.673 distro             1.7.0
#21 1.674 drsclient          0.2.3
#21 1.674 exceptiongroup     1.1.2
#21 1.675 fastavro           1.8.2
#21 1.675 frozenlist         1.4.0
#21 1.676 gdcdictionary      1.2.0
#21 1.676 gen3               4.20.0
#21 1.677 h11                0.14.0
#21 1.677 httpcore           0.16.3
#21 1.678 httpx              0.23.3
#21 1.679 humanfriendly      10.0
#21 1.679 idna               3.4
#21 1.680 indexclient        2.2.1
#21 1.680 jsonschema         3.2.0
#21 1.681 marshmallow        3.20.1
#21 1.681 multidict          6.0.4
#21 1.682 mypy-extensions    1.0.0
#21 1.683 numpy              1.25.1
#21 1.683 packaging          23.1
#21 1.684 pandas             1.5.3
#21 1.684 pip                23.2
#21 1.685 PyGObject          3.42.1
#21 1.685 pypfb              0.5.0
#21 1.686 pyrsistent         0.19.3
#21 1.687 python-dateutil    2.8.2
#21 1.687 python-json-logger 0.1.11
#21 1.688 pytz               2023.3
#21 1.688 PyYAML             5.3.1
#21 1.689 requests           2.31.0
#21 1.690 rfc3986            1.5.0
#21 1.690 setuptools         68.0.0
#21 1.691 six                1.16.0
#21 1.691 sniffio            1.3.0
#21 1.692 ssh-import-id      5.11
#21 1.692 tomlkit            0.11.8
#21 1.693 tqdm               4.65.0
#21 1.694 typing_extensions  4.7.1
#21 1.694 typing-inspect     0.9.0
#21 1.695 urllib3            1.26.16
#21 1.695 wheel              0.37.1
#21 1.696 xmltodict          0.13.0
#21 1.696 yarl               1.9.2
#21 1.697 yq                 3.2.2
#21 DONE 2.8s
```#21 [17/21] RUN pip list
#21 sha256:162d49c60fac4a67a109e9d6dd8243f8f448566422ae41d83e8f66ad03464887
#21 1.661 Package            Version
#21 1.662 ------------------ --------
#21 1.663 aiofiles           0.8.0
#21 1.663 aiohttp            3.8.5
#21 1.664 aiosignal          1.3.1
#21 1.665 anyio              3.7.1
#21 1.665 argcomplete        3.1.1
#21 1.666 async-timeout      4.0.2
#21 1.667 asyncio            3.4.3
#21 1.667 attrs              23.1.0
#21 1.668 backoff            1.11.1
#21 1.668 cdislogging        1.1.1
#21 1.669 certifi            2023.5.7
#21 1.669 charset-normalizer 3.2.0
#21 1.670 click              7.1.2
#21 1.671 crcmod             1.7
#21 1.671 dataclasses-json   0.5.13
#21 1.672 dbus-python        1.2.18
#21 1.672 dictionaryutils    3.0.2
#21 1.673 distro             1.7.0
#21 1.674 drsclient          0.2.3
#21 1.674 exceptiongroup     1.1.2
#21 1.675 fastavro           1.8.2
#21 1.675 frozenlist         1.4.0
#21 1.676 gdcdictionary      1.2.0
#21 1.676 gen3               4.20.0
#21 1.677 h11                0.14.0
#21 1.677 httpcore           0.16.3
#21 1.678 httpx              0.23.3
#21 1.679 humanfriendly      10.0
#21 1.679 idna               3.4
#21 1.680 indexclient        2.2.1
#21 1.680 jsonschema         3.2.0
#21 1.681 marshmallow        3.20.1
#21 1.681 multidict          6.0.4
#21 1.682 mypy-extensions    1.0.0
#21 1.683 numpy              1.25.1
#21 1.683 packaging          23.1
#21 1.684 pandas             1.5.3
#21 1.684 pip                23.2
#21 1.685 PyGObject          3.42.1
#21 1.685 pypfb              0.5.0
#21 1.686 pyrsistent         0.19.3
#21 1.687 python-dateutil    2.8.2
#21 1.687 python-json-logger 0.1.11
#21 1.688 pytz               2023.3
#21 1.688 PyYAML             5.3.1
#21 1.689 requests           2.31.0
#21 1.690 rfc3986            1.5.0
#21 1.690 setuptools         68.0.0
#21 1.691 six                1.16.0
#21 1.691 sniffio            1.3.0
#21 1.692 ssh-import-id      5.11
#21 1.692 tomlkit            0.11.8
#21 1.693 tqdm               4.65.0
#21 1.694 typing_extensions  4.7.1
#21 1.694 typing-inspect     0.9.0
#21 1.695 urllib3            1.26.16
#21 1.695 wheel              0.37.1
#21 1.696 xmltodict          0.13.0
#21 1.696 yarl               1.9.2
#21 1.697 yq                 3.2.2
#21 DONE 2.8s

**Review by k-burt-uch — APPROVED** [review]

**Pull request merged**